tests/suites/test_suite_ssl.function

```diff
@@ -6,13 +6,42 @@
 #include <mbedtls/certs.h>
 #include <mbedtls/timing.h>
 
+
+typedef struct log_pattern
+{
+    const char *pattern;
+    size_t *counter;
+} log_pattern;
+
+/*
+ * This function can be added as a debug output for log messages from mbedtls
```

Comment:
This is a bit unclear, I would rather state that: 
```suggestion
 * This function can be passed to mbedtls to receive output logs from it. In this case, it will count the instances of a log_pattern in the received logged messages.
```